app/Http/Controllers/TaskController.php

```diff
@@ -135,4 +135,37 @@ public function sort(Request $request)
         return $collectionTasks;
     }
 
+    public function getArchivedTasks(Request $request)
+    {   
+        $tasks =  ($this->taskService->all());
+         return response()->json($tasks);
+    }
+
+
+    public function getArchivedTasksById(Request $request,$id)
+    {           
+        $tasks = ($this->taskService->find($id));
+         return response()->json($tasks);
+    }
+
+    public function archiveTask(Request $request, $id)
```

Comment:
I really don't understand why you put this method here. What you are to work on has no business with creating a task or what looks like it. Please understand the issue before writing your code.